vendor/github.com/xshellinc/tools/constants/device.go

```diff
@@ -20,6 +20,7 @@ const (
 	DEVICE_TYPE_EDISON     = "edison"
 	DEVICE_TYPE_NANOPI     = "nano-pi"
 	DEVICE_TYPE_BEAGLEBONE = "beaglebone"
+	DEVICE_TYPE_COLIBRI    = "colibri iMX6"
```

Comment:
__[golint]__
* [golint] don't use ALL_CAPS in Go names; use CamelCase

<!-- Comment by [SideCI](https://sideci.com) -->